src/main/io/spektrum_rssi.c

```diff
@@ -144,7 +144,7 @@ void spektrumHandleRSSI(volatile uint8_t spekFrame[]) {
 #ifdef USE_SPEKTRUM_RSSI_PERCENT_CONVERSION
             // Do an dBm to percent conversion with an approxatelly linear distance
             // and map the percentage to RSSI RC channel range
-            spekChannelData[rssi_channel] = (uint16_t)(map(dBm2range (rssi),
+            spekChannelData[rssi_channel] = (uint16_t)(map(dBm2range (constrain(rssi, SPEKTRUM_RSSI_MIN, SPEKTRUM_RSSI_MAX)),
```

Comment:
Actually, looking at this, it makes more sense to move the `constrain()` to the start of `dBm2range ()`.

Reply:
updated now...